website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-improvements.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Run History usability and design improvements"
+sidebar_label: "Update: Run History improvements"
+sidebar_position: 4
+tags: [May-25-2023, Scheduler]
+---
+
+New usability and design improvements to the **Run History** dashboard in dbt Cloud are now available. These updates allow people to discover the information they need easier by reducing the number of clicks, surfacing more relevant information, keeping people in the flow state, and designing the look and feel to be more intuitive to use.   
```

Comment:
```suggestion
New usability and design improvements to the **Run History** dashboard in dbt Cloud are now available. These updates allow people to discover the information they need more easily by reducing the number of clicks, surfacing more relevant information, keeping people in the flow state, and designing the look and feel to be more intuitive to use.   
```

Reply:
"more easily" is an adverb modifying "discover"

Reply:
great catch, thank you, @schottj !